extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -189,7 +189,7 @@ export class AmpSlideScroll extends BaseSlides {
   /** @override */
   mutatedAttributesCallback(mutations) {
     const slide = mutations['slide'];
-    if (slide) {
+    if (slide !== undefined) {
```

Comment:
We also want to support strings here so I think `type#isFiniteNumber` wouldn't suffice.